bin/run

```diff
@@ -4,12 +4,21 @@
 use Symfony\Component\Console\Input\ArgvInput;
 use OpenEuropa\TaskRunner\TaskRunner;
 
-if (file_exists(__DIR__.'/../vendor/autoload.php')) {
-    $classLoader = require __DIR__.'/../vendor/autoload.php';
-} elseif (file_exists(__DIR__.'/../../../autoload.php')) {
-    $classLoader = require __DIR__ . '/../../../autoload.php';
+// If we're running from phar load the phar autoload file.
+$pharPath = \Phar::running(true);
+if ($pharPath) {
+    $autoloaderPath = "$pharPath/vendor/autoload.php";
+} else {
```

Comment:
Out of scope from the current PR, serving it as a Phar has implications we need to consider before having this in.